tests/sentinel/tests/05-manual.tcl

```diff
@@ -1,18 +1,26 @@
 # Test manual failover
 
-source "../tests/includes/init-tests.tcl"
+proc test_sentinel_failover {type master_id} {
 
 foreach_sentinel_id id {
     S $id sentinel debug info-period 2000
     S $id sentinel debug default-down-after 6000
     S $id sentinel debug publish-period 1000
 }
 
-test "Manual failover works" {
+set val 0
+
+test "Manual failover works - $type" {
+    R $master_id del foo
     set old_port [RPort $master_id]
     set addr [S 0 SENTINEL GET-PRIMARY-ADDR-BY-NAME mymaster]
     assert {[lindex $addr 1] == $old_port}
 
+    # Rename the FAILOVER command so that we can fallback to REPLICAOF NO ONE.
+    if {$type == "legacy"} {
+        S 0 SENTINEL SET mymaster rename-command FAILOVER FAILOVER_ERR
```

Comment:
Rename to ERR? it's not an error.. Maybe FAILOVER_OLD, FAILOVER_ORIGINAL or FAILOVER_RENAMED?

Reply:
This is just to rename to a non-existent name, so that when sentinel sends it to primary, it will return an Unknown command error, something like that.

Reply:
i will rename it to other thing to mention the non-existent thing.

Reply:
rename to FAILOVER_NONEXISTENT?